CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 
 ## Unreleased - 2017-??-??
```

Comment:
If we merge this PR, reader cannot understand which version will release following `Unreleased` fixes.
I think it's better to merge not `release-3.1` branch but the tagged commit as `3.1.1`.